data/widgets/card/title.ui

```diff
@@ -50,6 +52,7 @@
           <object class="EknThemeableImage" id="after">
             <property name="visible">True</property>
             <property name="valign">center</property>
+            <property name="vexpand">True</property>
```

Comment:
This causes some unpexpected regressions in other templares:

![virtualbox_endlessos_22_05_2017_10_58_20](https://cloud.githubusercontent.com/assets/481418/26315174/4668968c-3ede-11e7-9e43-ee22726f3942.png)

![virtualbox_endlessos_22_05_2017_11_00_34](https://cloud.githubusercontent.com/assets/481418/26315184/4b93d784-3ede-11e7-8885-f25a0aef2917.png)

Can we confine these changes to the news YAML?